test/fixtures/_dummyTestingApp/about-us.html

```diff
@@ -0,0 +1 @@
+This is another dummy page.
```

Comment:
can we move these fixtures so they live under:

```
test/fixtures/services/static-snapshot-service/
```

that way there's a clear mapping to which test they are for